app/scripts/modules/netflix/ci/ci.filter.model.ts

```diff
@@ -1,10 +0,0 @@
-import {module} from 'angular';
-
-export class CiFilterModel {
-  public searchFilter: string;
-}
-
-export const CI_FILTER_MODEL = 'spinnaker.netflix.ci.filter.model';
-
-module(CI_FILTER_MODEL, [])
```

Comment:
Normally, yes.  In this case, the field is static so no `new`ing is necessary.  But I think you're right in that we should have a conversation about how we want to manage global state.  This is also what redux is for, right?